### DISCUSSION TITLE
Deadline extended for simulation phase!

### DISCUSSION THREAD

**Discussion by fpetric** [larics/icuas22_competition#58] [Announcements]
Dear Teams, 

we are extending the simulation phase by a week! New cut-off date for your submissions is **May 8th** AOE. Good luck!

**Comment by Thanmal** [comment]
Dear @fpetric ,
Is there an exact time after which the submissions will close on May 8th or do we have until May 8th, 23:59 (UTC+2) ?

**Reply by fpetric** [reply]
May 8th AOE (anywhere on Earth).

**Reply by fpetric** [reply]
Sorry, that may not be clear, the deadline is May 8th 23:59 AOE.
